src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -122,12 +126,21 @@ class NUnit extends ConventionTask {
         project.file(getReportFolder())
     }
 
+    File getLogFolderImpl() {
+        if (logFolder)
+            project.file(logFolder)
+        else
+            return null
+    }
+
     @OutputFile
     File getTestReportPath() {
         // for the non-default nunit tasks, ensure we write the report in a separate file
-        // TODO: Do we need to supply prefix when user has specified its own report file name?
-        def reportFileNamePrefix = name == 'nunit' ? '' : name
```

Comment:
@taomWTP Well, in the new change we won't have this prefix anymore by default. How about if the user *does not* specify the report file, we keep the previous behaviour? But even so, I don't see why this change is needed for this PR 🤔